qopt_best_practices/transpilation/preset_qaoa_passmanager.py

```diff
@@ -0,0 +1,48 @@
+"""Make a pass manager to transpile QAOA."""
+
+from typing import Any, Dict
+
+from qiskit.transpiler import PassManager
+from qiskit.transpiler.passes import HighLevelSynthesis, InverseCancellation
+from qiskit.transpiler.passes.routing.commuting_2q_gate_routing import (
+    FindCommutingPauliEvolutions,
+    Commuting2qGateRouter,
+)
+from qiskit.circuit.library import CXGate
+
+from qopt_best_practices.transpilation.qaoa_construction_pass import QAOAConstructionPass
+
+
+def qaoa_swap_strategy(config: Dict[str, Any]):
```

Comment:
This is just a suggestion, but maybe we could add pm to the name to indicate it's a pass manager? I would personally not connect this function name to a pm generator. 

```suggestion
def qaoa_swap_strategy_pm(config: Dict[str, Any]):
```